mackerel-plugin-jvm/lib/jvm.go

```diff
@@ -373,6 +376,8 @@ func Do() {
 	optJavaName := flag.String("javaname", "", "Java app name")
 	optPidFile := flag.String("pidfile", "", "pidfile path")
 	optTempfile := flag.String("tempfile", "", "Temp file name")
+	optKeyPrefix := flag.String("metric-key-prefix", "", "Metric key prefix")
+	optLabelPrefix := flag.String("metric-label-prefix", "", "Metric label prefix")
```

Comment:
I feel that `metric-key-prefix` and `metric-label-prefix` are a little verbose to have `-prefix` in their names. Actually they are not used as prefix strictly.
How do you think just `metric-key` and `metric-label`?

Reply:
They are common name within repository, so there is no strong reason to change.